pkg/admissioncontroller/webhook/admission/resourcesize/handler.go

```diff
@@ -105,6 +131,42 @@ func (h *Handler) handle(req admission.Request) error {
 	return nil
 }
 
+func (h *Handler) handleCountLimit(req admission.Request, log logr.Logger, requestedResource *metav1.GroupVersionResource, count *int64) error {
+	if req.Namespace != "" {
+		// We only want to restrict non-namespaced resources
+		// namespaced resources can be restricted by ResourceQuotas
+		return nil
```

Comment:
I think we should log that a configured count is ignored because of a namespaced object.